applications/nrf5340_audio/src/audio/streamctrl_unicast_server.c

```diff
@@ -0,0 +1,722 @@
+/*
+ * Copyright (c) 2018 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+#include "streamctrl.h"
+
+#include <stddef.h>
```

Comment:
I dont think these three #includes are needed. I also removed several other #includes for streamctrl_broadcast_sink